pyproject.toml

```diff
@@ -219,6 +219,7 @@ reportUnknownParameterType = false
 reportUnknownVariableType = false
 reportUnusedVariable = false
 reportPrivateUsage = false
+reportPrivateImportUsage = false
```

Comment:
So this becomes a problem, because `reportPrivateImportUsage = true` is on when people use `pyright` and `pandas-stubs` is installed.

See https://github.com/microsoft/pyright/issues/10248 for a discussion.  It looks like the pyright maintainer is going to consider reverting the change that created this issue, so if he does that, we can handle this PR later.

Reply:
Okay that is very insightful!
Let's hold on this PR before pyright clarifies the situation, for now the other PRs I have made have locked pyright version.

Reply:
@loicdiridollou pyright 1.1.400 has been released and they removed the thing that tests for private imports.  Can you update `pyproject.toml` in this PR so that we can get the CI to pass with 1.1.400?